build-support/mypy/mypy.ini

```diff
@@ -1,8 +1,7 @@
 [mypy]
-# TODO(#10131): Re-enable this plugin once we add support to v2 MyPy.
 # See: https://mypy.readthedocs.io/en/latest/extending_mypy.html#configuring-mypy-to-use-plugins
-# plugins =
-#   pants.mypy.plugins.total_ordering
+plugins =
+  mypy_plugins.total_ordering
```

Comment:
Moved to emphasize that this is an internal-only plugin. This does not get distributed to end-users.